addons/web_editor/static/src/js/editor/odoo-editor/src/OdooEditor.js

```diff
@@ -2231,7 +2231,7 @@ export class OdooEditor extends EventTarget {
             // Do not apply commands out of the editable area.
             return false;
         }
-        if (!sel.isCollapsed && BACKSPACE_FIRST_COMMANDS.includes(method)) {
+        if (!sel.isCollapsed && (BACKSPACE_FIRST_COMMANDS.includes(method) || this.powerbox.isOpen)) {
```

Comment:
@deso-odoo Oh, wow, ok, that's weird. @lebl-odoo Are we sure this is what we want though? In a way I always thought it kind of weird that writing "/head" _over_ some content worked to transform it to a header, but in another way I always thought it was intended and was kind of a neat trick.

Reply:
Hi @dmo-odoo & @deso-odoo ,

The main idea is when I select a text and type "/", like any other character, I would like to erase the content to make room for my new content (here, the action of the powerbox).

If you want to transform the actual "block" (e.g. paragraph), then just type "/" + <action> to apply it on it (e.g. "Hello world /heading 1" + apply will transform the paragraph into a Heading 1).

We should stick to such a behavior to not mislead users with custom and weird tricks :D

Reply:
@deso-odoo I agree with @lebl-odoo on the end result. However, I still think `_applyRawCommand` is a weird place to alter the behavior according to the powerbox. Previously I advocated to move the change in the code that inserted the separator, but now that we agree that it should affect all powerbox commands then I think the change should be global to the powerbox, but not global to every command like it is the case in the suggested fix here. Could you have a look on how it could be done in html_editor in master and maybe that can inform the right place to make the change in web_editor?

Reply:
@dmo-odoo Initially, I attempted to solve the problem by following the approach in the master branch. In `html_editor`, when inputting "/", a savepoint is created by storing `this.currentSteps` in `draftMutations`, and `revert` is called on these mutations before applying the command. While this works fine in `web_editor`, but it causes an issue in the Knowledge Tour where tooltips are involved (Runbot - https://runbot.odoo.com/runbot/batch/1657520). The tooltip mutation results in two tooltips being displayed simultaneously.

According to the task specifications, we need to remove the selected text first and then apply the command. Taking these factors into account, I determined that `applyRawCommand` was more appropriate place for this operation.

cc - @deso-odoo

Reply:
@dmo-odoo Yes, this issue does not occur in version 18.0.

Reply:
@adch-odoo Alright then, I'll merge it. Thanks!